byos/devops/GH.md

```diff
@@ -21,8 +21,9 @@ This lab deploys to a single resource group within an Azure subscription. To dep
 
 Each OpenHack team will create **their own GitHub team within the GitHub organization that has already been created for all DevOps OpenHacks**.
 
-- Coaches should be added to the GitHub organization prior each OpenHack. This is managed by the OpenHack program team.
-- External coaches may need assistance accessing the GitHub organization. They should contact the OpenHack program team for support before the OpenHack delivery.
+- Tech Lead should be added to the GitHub organization prior each OpenHack. This is managed by the OpenHack Program Team.
```

Comment:
Suggest making a combining the sentences (e.g. Tech Lead should be added to the GitHub organization prior to each Open Hack by the Open Hack Program Team)